tests/subsys/tracing/tracing_sys/src/main.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
```

Comment:
Please change the year to 2021. So does  "tests/subsys/tracing/tracing_sys/src/tracing_string_format_test.c: line 2"

Reply:
Done, thanks